django/joist/admin.py

```diff
@@ -1,6 +1,6 @@
 from django.contrib import admin
 
-from core.models import Blob
+from .models import Blob
```

Comment:
@sgratzl Are the relative imports to avoid Django's syspath magic or to make this easier to separate as a separable module outside of an app?

Reply:
in this case the Blob model actually moved from `core.models` to `joist.models`. I personally prefer relative imports within a module since then it is easier to rename the whole module